suites/nautilus/rados/sanity_rados.yaml

```diff
@@ -96,6 +96,15 @@ tests:
       polarion_id: CEPH-9924
       desc: rewrite a known omap item of a replica and check list-inconsistent-obj
 
+   - test:
+       name: Auto trim of ceph osd maps
+       polarion-id: CEPH-83573514
+       module: test_bz_1829646.py
```

Comment:
is bug present in luminous too? if so please add it to luminous sanity_rados_lvm suite too